packages/docusaurus-theme-classic/codeTranslations/fa.json

```diff
@@ -1,56 +1,110 @@
 {
   "theme.AnnouncementBar.closeButtonAriaLabel": "بستن",
+  "theme.AnnouncementBar.closeButtonAriaLabel___DESCRIPTION": "The ARIA label for close button of announcement bar",
```

Comment:
Can you remove those `___DESCRIPTION` attributes please? They should not be added

Reply:
Done. Thanks